### PR TITLE
Use correct UIFW config year when showing the UIFW indicator

### PR DESCRIPTION
**Pull request by michaelglenister**
Issue: https://trello.com/c/SPFwaZvB/520-uifw-years-config-why-is-uifw-referring-to-the-latest-year-when-its-year-config-has-not-been-modified-is-it-using-the-wrong-year

image

### CODE REVIEW

File: scorecard/profile_data/indicators/series.py [hunk 1]
```diff
@@ -12,3 +12,18 @@ def get_muni_specifics(cls, api_data):
...
+        results = api_data.results
+        years = api_data.uifw_years
+
+        return {
+            "result_type": cls.result_type,
+            "values": cls.get_values(years, results),
+            "ref": api_data.references[cls.reference],
+            "last_year": years[0] if len(years) > 0 else None,
+            "formula": cls.formula,
+        }
```
[the remark this answers is not in the document]
Comment (jbothma) [reply scorecard/profile_data/indicators/series.py:29]: the thinking being that it's nice for these indicators to have a common ancestor - we're saying they're the same _except_ for this method.

I didn't realise this was the only method in the class, but I think it's still nice for them to have the common ancestore so they can share any future common code.

[the remark this answers is not in the document]
Comment (jbothma) [reply scorecard/profile_data/indicators/series.py:29]: you know what, in other languages like Java, these would be two classes implementing an interface.

[the remark this answers is not in the document]
Comment (michaelglenister) [reply scorecard/profile_data/indicators/series.py]: Yeah, that's a much more maintainable way to do it. Updated the PR

**Comment by jbothma** [comment]
Please confirm this change using a different year for UIFW than the other years in settings passed in https://github.com/OpenUpSA/municipal-data/blob/master/scorecard/tests/indicators/test_uifw_expenditure.py#L40.

Should be able to verify this change using a test by changing just one of these years and seeing the other years change in the resulting data, right?

**Pull request merged**